app/Http/Controllers/AdminController.php

```diff
@@ -76,4 +77,57 @@ public function saveImage($request)
         return $image_name;
 
     }
+    public function deleteImage($image_name){
+        $image_path = self::LOCAL_STORAGE_FOLDER.$image_name;
+
+        if(Storage::disk('local')->exists($image_path)){
+            Storage::disk('local')->delete($image_path);
+        }
+    }
+
+    public function editEvent(Event $event)
+    {
+        $eventLevels = $event->levels->pluck('id')->toArray();
+
+        return view('admin.events.edit')
+            ->with('event',$event)
+            ->with('eventLevels',$eventLevels);
+
+    }
+
+    public function updateEvent($id, Request $request)
+    {
+        $request->validate([
```

Comment:
please refer to this
https://kredo-intern-based.slack.com/archives/C04RD83NBKJ/p1686828838684099?thread_ts=1686804800.473949&cid=C04RD83NBKJ